src/video.cpp

```diff
@@ -999,6 +999,52 @@ namespace video {
     },
     LIMITED_GOP_SIZE | PARALLEL_ENCODING | SINGLE_SLICE_ONLY | NO_RC_BUF_LIMIT
   };
+
+  static encoder_t v4l2m2m {
+    "V4L2M2M"sv,
+    std::make_unique<encoder_platform_formats_avcodec>(
+      AV_HWDEVICE_TYPE_NONE, AV_HWDEVICE_TYPE_NONE,
+      AV_PIX_FMT_NONE,
+      AV_PIX_FMT_YUV420P, AV_PIX_FMT_YUV420P10,
+      nullptr),
+    {
+      // Common options
+      {},
+      // SDR-specific options
+      {},
+      // HDR-specific options
+      {},
+      // Fallback options
+      {},
+      std::make_optional<encoder_t::option_t>("qp"s, &config::video.qp),
```

Comment:
Does the encoder really not support CBR/VBR bitrate control? QP shouldn't be provided if CBR or VBR is available.

Reply:
Probably, I was just copying what the others did as a first step. I'll give it a try.